src/perftest_parameters.h

```diff
@@ -191,17 +191,17 @@
 #define CYCLES	"cycles"
 #define USEC	"usec"
 /* The format of the results */
-#define RESULT_FMT		" #bytes     #iterations    BW peak[MB/sec]    BW average[MB/sec]   MsgRate[Mpps]"
+#define RESULT_FMT		" #bytes     #iterations    BW peak[MB/sec]    BW average[MB/sec]   MsgRate[Mpps]       BW min[MB/sec]"
```

Comment:
Can you please keep original report if min BW wasn't requested?

Reply:
Sure, I was trying to avoid that as it's touching a lot of code :)